css-brochure-form/styles.css

```diff
@@ -0,0 +1,59 @@
+form {
+  width: 50%;
+  margin: auto;
+}
+
+input {
+  width: 100%;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-left: 10px;
+  font-size: 18px;
+  border-radius: 7px;
+  border: 1px solid gray;
+  margin-top: 5px;
+  margin-bottom: 20px;
+}
+
+h2.centered {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
+  color: rgb(236 93 87);
+  text-align: center;
+}
+
+label.bold {
+  padding-left: 10px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
```

Comment:
This line is only necessary for the `@font-face` rules to include the fonts in the document. In order to use the font, you just need to set `font-family`